Dockerfile

```diff
@@ -1,35 +1,28 @@
-FROM debian
-MAINTAINER Andreas Stenius git@astekk.se
+FROM alpine:3.4
+MAINTAINER Marco Wessel <marco@driebit.nl>
 
-ENV DEBIAN_FRONTEND noninteractive
-
-# httpredir.debian.org/debian fails too much, so replace it with a fixed mirror.
-RUN echo \
-   'deb ftp://ftp.nl.debian.org/debian/ jessie main\n \
-    deb ftp://ftp.nl.debian.org/debian/ jessie-updates main\n \
-    deb http://security.debian.org jessie/updates main\n' \
-    > /etc/apt/sources.list
+ADD . /opt/zotonic
+WORKDIR /opt/zotonic
 
-ADD https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb .
 
-RUN apt-get clean \
-    && dpkg -i erlang-solutions_1.0_all.deb \
-    && apt-get update \
-    && apt-get install -y --no-install-recommends erlang build-essential ca-certificates imagemagick inotify-tools libnotify-bin wget git \
-    && rm -rf /var/lib/apt/lists/*
+# Note: dumb-init and gosu are pulled from edge; remove that when upgrading to an alpine release that
+# includes those packages.
+RUN apk add --virtual build-deps --no-cache ca-certificates wget curl make gcc musl-dev g++ git \
+        && apk add --no-cache bash imagemagick \
+	&& apk add --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ dumb-init gosu \
+           erlang erlang-inets erlang-compiler erlang-crypto erlang-mnesia erlang-ssl erlang-stdlib erlang-public-key erlang-tools erlang-dev erlang-asn1 erlang-syntax-tools erlang-eunit erlang-parsetools erlang-snmp erlang-sasl erlang-xmerl erlang-erl-interface \
+        && DEBUG=1 make \
+        && apk del build-deps
```

Comment:
While we want to remove build-deps for production containers, don’t we need to keep them for development containers so that changes in `user/{modules|sites}` can be compiled?